alpha/apps/kaltura/lib/multiLingualUtils.php

```diff
@@ -132,7 +132,7 @@ public static function addFieldMappingToMultiLangMapping(&$multiLingualMapping,
 		{
 			foreach ($valueToAdd as $languageKey => $languageValue)
 			{
-				if ($multiLingualMapping[$field][$languageKey] != $languageValue)
+				if (!isset($multiLingualMapping[$field][$languageKey]) || $multiLingualMapping[$field][$languageKey] != $languageValue)
```

Comment:
for sure we have $multiLingualMapping[$field]?
if not we will still get notice in the next line

Reply:
i think that if it doesnt exist, the next line creates it. the next line doest lookup the value it sets it